feat(Spinner): remove isSVG prop (#236)
patternfly/pf-codemods: packages/pf-codemods/README.md

<SimpleList isActive />
```

> ### spinner-svg-default [(#8616)](https://github.com/patternfly/patternfly-react/pull/8616)

Reviewer: Nit: since we're updating what the codemod does, we could update the file names/heading here to refer to the fact that the SVG prop is being removed.

> ### spinner-remove-issvg [(#8616)](https://github.com/patternfly/patternfly-react/pull/8616)

Reviewer 2: ```suggestion
### spinner-remove-isSvg [(#8616)](https://github.com/patternfly/patternfly-react/pull/8616)
```

> We've updated the Spinner to only be an svg and have removed the isSVG prop.

Author: I thought "only be" would make it more clear that it cannot be anything else. When I hear "default", it would make me think there is still a way to change it.

> We've updated the Spinner to only be an svg and have removed the isSVG prop.

Reviewer 2: Ahh that certainly makes sense and you make a great point. In that case, what do you think of something like:
```suggestion
We've updated the Spinner to exclusively use an SVG, and have removed the isSVG prop.
```
